src/models/Contact.ts

```diff
@@ -50,6 +50,7 @@ export default class Contact {
   @Column(() => Password)
   password!: Password;
 
+  /** @deprecated This is used in the legacy frontend, use the `ContactMfa` model / table instead for the new frontend */
   @Column(() => OneTimePassword)
   otp!: OneTimePassword;
```

Comment:
I think let's actually delete this and remove all the references to it

Reply:
Happy to take this on or pair on it if you want as the legacy app is pretty horrible in places.

Reply:
Are you sure? I don't want to break anything, since you said that the old frontend is still being used?

Reply:
We can defo remove it and not break anything, but it will involve deleting a fair amount of old code. Once the user is logged in the session is shared between the new frontend and legacy app, so we just need to remove anything that involves authentication from the legacy app (e.g `src/apps/otp`).